### PR TITLE
Switch image builds y-axis on Gitpod/Overview dashboard

### PR DESCRIPTION
**Pull request by WVerlaek**
## Description

Use the right y-axis for image build workspaces in the Grafana overview dashboard. Makes these workspaces stand out more, as they're usually hidden due to the amount of regular workspaces. Would let us detect lingering image build workspaces.

In an attempt to make it more visible that image builds use the right y-axis, made the lines dashed and staircased, but open to other suggestions 🙏 

See https://grafana.gitpod.io/d/y5txMP2Vk/gitpod-overview-duplicate-wouter?from=now-1h&to=now

## How to test

See copy of overview dashboard with these changes here: https://grafana.gitpod.io/d/y5txMP2Vk/gitpod-overview-duplicate-wouter?from=now-1h&to=now

## Release Notes

```release-note
NONE
```

## Werft options:

- [ ] /werft with-local-preview
      If enabled this will build `install/preview`
- [ ] /werft with-preview
- [ ] /werft with-large-vm
- [ ] /werft with-integration-tests=all
      Valid options are `all`, `workspace`, `webapp`, `ide`, `jetbrains`, `vscode`, `ssh`

### CODE REVIEW

File: operations/observability/mixins/cross-teams/dashboards/gitpod-overview.json [hunk 1]
```diff
@@ -96,7 +96,8 @@
                 "steps": []
               },
               "unit": "none"
-            }
+            },
+            "overrides": []
```
Comment (WVerlaek) [inline operations/observability/mixins/cross-teams/dashboards/gitpod-overview.json:100]: Unfortunately duplicating the Overview dashboard and downloading its JSON also added a number of unrelated changes to the diff, I think these are changes from newer Grafana versions that haven't been persisted yet to git?

I've only really updated the "Running workspaces" panels.

**Review by sagor999 — APPROVED** [review]
